### PR TITLE
Bug(SC-698): Rename function  and natspec of sellFor to sellTo

### PR DESCRIPTION
**Pull request by Zitzak**
The function and natspec for sellFor were misleading, as we are not enabling selling for someone else, but sending the redeemed token to a specified address. The function name and comments have been updated to reflect this

### CODE REVIEW

File: src/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.sol [hunk 1]
```diff
@@ -230,7 +230,7 @@ contract FM_BC_Bancor_Redeeming_VirtualSupply_v1 is
         buyFor(_msgSender(), _depositAmount, _minAmountOut);
     }
 
-    /// @notice Redeem tokens on behalf of a specified receiver address. This function is subject
+    /// @notice Redeem tokens and directs the proceeds to a specified receiver address. This function is subject
```
Comment (marvinkruse) [inline src/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.sol]: `Redeem` is using the infinitive form, so `directs` should be `direct`

**Comment by marvinkruse** [comment]
Perfect, besides the tiny comment!

Edit: I resolved it myself to make this easier, if you don't like that just reserve it and publicly shame me :D

Otherwise, ignore the weird "expected checks" that don't run, this will work fine as soon as #608 is merged, as this changes that a bit.

File: src/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.sol (hunk 1 above)
Comment (marvinkruse) [reply src/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.sol]: Took the liberty of doing it myself, hope that's okay!

**Comment by Zitzak** [comment]
Thank you for the review and changing the code. COuld you still approve the PR so we can merge it later :) @marvinkruse

**Review by 0xNuggan — APPROVED** [review]

**Review by marvinkruse — APPROVED** [review]

**Pull request merged**